Adds a new traitor objective in celebration of open source goonstation

Manly-Dwarf opened the pull request:
## About the PR
Traitors are now tasked with stealing the Gooncode.
<!-- Describe your Pull Request here. What does it change? What other things could this affect? -->
_Inspired by_ classic HippieStation. https://github.com/HippieStationArchive/HippieStation13-1/pull/1941
## Why's this needed?
Yes.
<!-- Describe why you think this should be added to the game. -->
The gooncode's location varies from map to map. In some it's just in aux storage, in some it's in a wall in disposals, in others it's just in a crate in the mail router. You can also get it via crane if you're very lucky.

Manly-Dwarf commented on goonstation.dme:
```diff
@@ -46,6 +46,30 @@ var/datum/preMapLoad/preMapLoad = new
 #include "code\__secret.dm"
 
 // BEGIN_INCLUDE
+#include "_stdlib\__build.dm"
```
uh

ElephantSeal replied:
yeah I would want this added

karmaisblackandbluepilled commented:
yall gon make me prrt ah ha ah ha prrt ah ha ah pa ha prrrrt prrt yall gon make me ah ha prrt puh ah ha ah ha prt prt prt yall gon make me ha hab prrt ah ha ah prrt ah ha ah ha prrt prrt prrt pum ah ha seh ee seh ee pum ah ha don seh ee pum ah ha seh ee pum pum lemme show you how to scratch it.. pum ah ha seh eh pum ah ha don sa pum ah ha pum ah ha seh eh pum pum pum ah ha se eh se eh pum ah ha seh eh pum ah ha don sa pum ah ha se eh pum pum pum ah ha se he se eh either way destiny still arrives byeuah

Manly-Dwarf commented:
it wouldn't be quality without some breaking first

ZeWaka commented:
Also unsure if you even have permission to port this. Our license is incompatible with Hippie's.

Manly-Dwarf commented:
@flrsh on it, also contacted the original guy who made this.

Manly-Dwarf commented:
![gooncode](https://user-images.githubusercontent.com/33744849/78198158-aeaf7d00-7455-11ea-9d6e-80f6a4af8d22.png)
preview of item when used on a PDA ingame, for those who can't read the code

Manly-Dwarf commented:
The only thing from HippieStation that was truly ported was a sprite, which I have now changed to a completely "original" one (a harddisk given bee stripes) so this is no longer a "port" and license is no longer a concern.
Fixed the .dme, removed the gooncode from plushie code and icons, and added a little bit more to the madlibs.
